native/src/main/java/io/ballerina/stdlib/grpc/builder/syntaxtree/utils/MessageUtils.java

```diff
@@ -103,38 +103,59 @@ private MessageUtils() {
     private static Type getMessageType(Message message) {
         Record messageRecord = new Record();
         for (Field field : message.getFieldList()) {
-            switch (field.getFieldType()) {
-                case "string":
-                case "int":
-                case "float":
-                case "boolean":
-                    if (field.getFieldLabel() == null) {
+            if (field.getFieldLabel() == null) {
+                switch (field.getFieldType()) {
+                    case "string":
+                    case "int":
+                    case "float":
+                    case "boolean":
                         messageRecord.addBasicFieldWithDefaultValue(field.getFieldType(), field.getFieldName(),
                                 field.getDefaultValue());
-                    } else {
-                        messageRecord.addArrayFieldWithDefaultValue(field.getFieldType(), field.getFieldName());
-                    }
-                    break;
-                case "byte[]":
-                    messageRecord.addArrayFieldWithDefaultValue("byte", field.getFieldName());
-                    break;
-                case "Timestamp":
-                    messageRecord.addCustomFieldWithDefaultValue("time:Utc", field.getFieldName(), "[0, 0.0d]");
-                    break;
-                case "Duration":
-                    messageRecord.addCustomFieldWithDefaultValue("time:Seconds", field.getFieldName(), "0.0d");
-                    break;
-                case "Struct":
-                    messageRecord.addMapFieldWithDefaultValue(SyntaxTreeConstants.SYNTAX_TREE_VAR_ANYDATA,
-                            field.getFieldName());
-                    break;
-                default:
-                    if (field.getFieldLabel() == null) {
+                        break;
+                    case "byte[]":
+                        messageRecord.addArrayFieldWithDefaultValue("byte", field.getFieldName());
+                        break;
+                    case "Timestamp":
+                        messageRecord.addCustomFieldWithDefaultValue("time:Utc", field.getFieldName(), "[0, 0.0d]");
+                        break;
+                    case "Duration":
+                        messageRecord.addCustomFieldWithDefaultValue("time:Seconds", field.getFieldName(), "0.0d");
+                        break;
+                    case "Struct":
+                        messageRecord.addMapFieldWithDefaultValue(SyntaxTreeConstants.SYNTAX_TREE_VAR_ANYDATA,
+                                field.getFieldName());
+                        break;
+                    case "'any:Any":
+                        messageRecord.addCustomFieldWithDefaultValue("'any:Any", field.getFieldName(), "{}");
```

Comment:
I think we have to update the `protobuf` module with default values with this. Shall we create a task?.

Reply:
Default values were added in [[1]](https://github.com/ballerina-platform/module-ballerina-protobuf/pull/165).

[1] https://github.com/ballerina-platform/module-ballerina-protobuf/pull/165